### DISCUSSION TITLE
Automatic Backup Vs External file sync

### DISCUSSION THREAD

**Discussion by andriykrefer** [immich-app/immich#11011] [Q&A]
Hi,

I am currently analyzing if it is better to use the app "Automatic Backup" or an external file sync App with external libraries.
However, I could not find the current behavior of the backup on some edge cases, like what happens when:

- Already uploaded files are moved in the phone (from Camera to another album folder)
- Already uploaded files are deleted in the phone

Besides, that's the pros of each method I can think of:

- Automatic backup: Local videos on the phone are played locally and do not need to be downloaded (videos can be pretty huge, and that makes playing them always quick)
- External lib sync: Original file names and structure are kept

Regards

**Comment by MexHigh** [comment]
I would go with the built-in automatic backup. I think using an external library just for this might introduce unnecessary complexity in your setup. 

To answer your edge case questions:
- When an already uploaded file is moved locally, the immich app recognizes this as a new file. It will be uploaded again, however, it gets deduplicated after upload automatically so it does not appear twice in your timeline
- When you delete already uploaded photos locally, nothing happens. The image remains on your immich server and the immich app will show you the remote copy (this is the intended procedure to free up your local storage space without losing your photos, as they are already backed up)
